perf/perf_rawevents.py

```diff
@@ -45,7 +45,10 @@ def setUp(self):
         smm = SoftwareManager()
         detected_distro = distro.detect()
         self.distro_name = detected_distro.name
-        self.cpu_family = cpu.get_family()[5:]
+        processor = genio.read_file("/proc/cpuinfo")
+        for line in processor.splitlines():
+            if 'revision' in line:
+                self.rev = (line.split(':')[1].strip())
```

Comment:
@disgoel @PraveenPenguin can we move this part to common utils?
like get_cpu_family()?

Reply:
seems me good idea